<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstrumentInfomation</class>
 <widget class="QDialog" name="InstrumentInfomation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>仪器信息</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,1">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>使用记录</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="tableView">
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>残值曲线</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="wg_graph" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1,1,1,1,1,1">
     <item>
      <widget class="ProgressWater" name="widget" native="true"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>  设备名称:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_name">
         <property name="text">
          <string>惠普打印机</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>  设备编号:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_number">
         <property name="text">
          <string>78441528</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>  型号规格:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_guige">
         <property name="text">
          <string>Z50</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>  设备价格:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_price">
         <property name="text">
          <string>5999</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>  入库时间:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_time">
         <property name="text">
          <string>2014-12-16 09:28</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>  设备条码:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_id">
         <property name="text">
          <string>YQ1412160925800740</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgressWater</class>
   <extends>QWidget</extends>
   <header location="global">progresswater.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
